src/styles/main.css

```diff
@@ -0,0 +1,188 @@
+body {
+  margin: 0;
+  font-family: Arial, Baghdad, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 720px;
```

Comment:
The width of the calendar is larger than required by the task. If each day is represented by a 100px square and there are 7 columns with a 1px gap, the width should be 707px (7*100 + 6*1).

Reply:
Valid comment, but probably something is still wrong with your padding